app/model/Payment/ReadModel/Queries/PreparedPairedPaymentsQuery.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Model\Payment\ReadModel\Queries;
+
+use Model\Payment\BankAccount\BankAccountId;
+
+/**
+ * @see PreparedPaymentsQueryHandler
+ */
+final class PreparedPairedPaymentsQuery
```

Comment:
Tady bych radši místo `PairedPaymentsQuery` a `PreparedPairedPaymentsQuery` použil obecnější query, kde budeme předávat `State` pro filtrování plateb podle stavu.

Reply:
muzu to zmenit na `State` ale ten paired není stated, takze stejne zustanou 2 proudy